tests/edvinas/Calculator.java

```diff
@@ -0,0 +1,57 @@
+package edvinas;
+
+class Calculator{
+    public static void main(String[] args) {
+
+        System.out.println(calculator("5+5") + factorial(5));
+
+//        System.out.println(factorial(5));
+
+    }
+
+    public static int calculator(String input) {
```

Comment:
Very hard to read, maybe you could split stuff by blank lines or maybe into functions?

Reply:
Better ?